.github/workflows/build.yaml

```
@@ -16,11 +16,27 @@ jobs:
+        with:
+          path: |
+            ~/.cargo
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-
+
       - name: Setup Rust toolchain
-        run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
-      - name: Build workspace (excluding openrpc-testgen/openrpc-testgen-runner)
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)